ui/src/components/header/HeaderNotice.vue

```diff
@@ -108,8 +108,12 @@ export default {
     this.$store.watch(
       (state, getters) => getters.headerNotices,
       (newValue, oldValue) => {
-        if (oldValue !== newValue && newValue !== undefined) {
-          this.notices = newValue.reverse()
+        if (newValue !== undefined) {
+          return
+        }
+        const reversedNewValue = newValue.reverse()
+        if (oldValue !== reversedNewValue) {
+          this.notices = reversedNewValue
```

Comment:
this feels like a bit of a stretch. Could we just check for notifications length @shwstppr ? (No new velue added should suffice, right? old values will not be removed just at a whim)

Reply:
@shwstppr , can you explain ^?

Reply:
@DaanHoogland I'm not sure about the length check at the moment. I've just fixed the existing check therefore